xmps/g.dat.txt

```diff
@@ -0,0 +1,96 @@
+# wavelength       sensitivity      assumed
```

Comment:
Let's put this and all data files into

xmps/data

and make separate folders within there

and a README.md file that explains 
what is in each folder.